CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
   snapshot.
 - Added a new API call, `PUT /snapshot/load`, for loading a snapshot.
 - Added metrics for the vsock device.
+- Added devtool strip command to strip release binaries out of DWARF sections.
```

Comment:
hm, "strip the binary out of?" We are not removing DWARF sections, right? Should we?

Anyhow, I'd change this to something more obvious "Added `devtool strip` command which removes debug symbols from the release binaries."